test/annotation_layer_builder_overrides.css

```
@@ -30,12 +30,19 @@
 }
 
 .annotationLayer :is(.linkAnnotation, .buttonWidgetAnnotation.pushButton) > a,
-.annotationLayer .popupTriggerArea {
+.annotationLayer .popupTriggerArea:last-child::after {
   opacity: 0.2;
   background: rgba(255, 255, 0, 1);
   box-shadow: 0 2px 10px rgba(255, 255, 0, 1);
 }
 
+.annotationLayer .popupTriggerArea:last-child::after {
+  display: block;
+  width: 100%;
+  height: 100%;
+  content: "";
+}
+
 .annotationLayer :is(.popupAnnotation, .popupWrapper) {
```

There's no longer any `.popupWrapper` rule being used, so assuming that this block is still necessary this could be simplified as follows.
```suggestion
.annotationLayer .popupAnnotation {
```